Rakefile

```diff
@@ -65,7 +70,8 @@ namespace :build do
     transform = RUBY_PLATFORM =~ /darwin/ ? "-s " : "--transform s"
     transform << "',^,#{tar_basename}/,'"
 
-    `tar -C ../manageiq #{transform} -X #{exclude_file} -hcvzf #{tarball} .`
+    # Everything from */tmp/* should be excluded, except for tmp/cache/sti_loader.yml
+    `tar -C ../manageiq #{transform} --exclude-tag='cache/sti_loader.yml' -X #{exclude_file} -hcvzf #{tarball} .`
```

Comment:
@simaishi this is great. ship it
Just making sure @matthewd understands how this is being used.

putting files into /tmp are typically temp files.
But it looks like we are precompiling assets and precompiling dependencies.
So just want that to be known as we further evolve the design.

again. not a concern. :shipit: